validation.py

```diff
@@ -90,7 +90,7 @@ def onepop_constantN_slim2(out_dir, seed):
 def onepop_constantN_slim3(out_dir, seed):
     """
     Single population with constant population size.
-    Time is rescaled by a factor of 10.
+    Time/Ne are rescaled by a factor of 10.
```

Comment:
I read this as "time divided by Ne" initially. An "and" seems less ambiguous.

Reply:
Yeah, good call. Fixed.